nixos/modules/programs/light.nix

```diff
@@ -17,11 +18,59 @@ in
           and udev rules granting access to members of the "video" group.
         '';
       };
+
+      brightnessKeys = {
+        enable = mkOption {
+          type = types.bool;
+          default = false;
+          description = ''
+            Whether to enable brightness control with keyboard keys.
+
+            This is mainly useful for minimalistic (desktop) environments. You
+            may want to leave this disabled if you run a feature-rich desktop
+            environment such as KDE, GNOME or Xfce as those handle the
+            brightness keys themselves. However, enabling brightness control
+            with this setting makes the control independent of X, so the keys
+            work in non-graphical ttys, so you might want to consider using this
+            instead of the default offered by the desktop environment.
```

Comment:
Good docs.

Since this is a thing NixOS offers as an abstraction, you should leave some breadcrumbs on how it is implemented.

Reply:
Do you mean adding a mention to actkbd?

For comparison, here is the current `sound.mediaKeys.enable` option description:

> Whether to enable volume and capture control with keyboard media keys.
>
> You want to leave this disabled if you run a desktop environment like KDE, Gnome, Xfce, etc, as those handle such things themselves. You might want to enable this if you run a minimalistic desktop environment or work from bare linux ttys/framebuffers.
>
> Enabling this will turn on `services.actkbd`.

Reply:
Mentioned actkbd now.